src/components/StudyInfo/StudyInfo.js

```diff
@@ -127,22 +137,43 @@ const StudyInfo = ({studyNode, setShowModal, unlinkProject}) => {
             </Segment>
           </Segment.Group>
         </Grid.Column>
-
         <Grid.Column
           mobile={16}
           tablet={16}
           computer={longDescription ? 16 : 8}
         >
-          <Segment padded>
+          <Segment
+            padded
+            attached={
+              studyNode.description && studyNode.description.length > 3500
```

Comment:
Now that I think of it, it may be better to not try to do the read more/less based on length as we allow images, tables, and other nodes that may have inconsistent line height.
Instead, can we just always show the read more/less regardless of the content length which simply switches off the max height class?